packages/core/src/domain/configuration/transportConfiguration.ts

```diff
@@ -21,18 +21,24 @@ export interface ReplicaConfiguration {
 }
 
 export function computeTransportConfiguration(initConfiguration: InitConfiguration): TransportConfiguration {
+  const { site = INTAKE_SITE_US1 } = initConfiguration
```

Comment:
**💬 suggestion:** ‏`initConfiguration` is directly the object provided by the user. `initConfiguration.site` is typed as `string | undefined`, but in practice it can be anything, like an empty string.

Changing from `initConfiguration.site || INTAKE_SITE_US1` to `{ site = INTAKE_SITE_US1 } = initConfiguration` could be a breaking change in that case.

Suggestion:
```suggestion
  const site = initConfiguration.site || INTAKE_SITE_US1
```